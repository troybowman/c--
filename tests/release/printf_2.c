../decl/printf_2.c