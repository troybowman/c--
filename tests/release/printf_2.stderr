error, line 1: ellipsis "..." notation is is only valid when declaring builtin function: extern void printf(char format[], ...);
syntax error, line: 2
syntax error, line: 3
error, line 4: ellipsis "..." notation is is only valid when declaring builtin function: extern void printf(char format[], ...);
syntax error, line: 5
error, line 6: ellipsis "..." notation is is only valid when declaring builtin function: extern void printf(char format[], ...);
error, line 7: ellipsis "..." notation is is only valid when declaring builtin function: extern void printf(char format[], ...);
error: function printf is defined at line 9 but is declared extern at line 8
c-- returned a nonzero exit code
